src/data/Storage.ts

```diff
@@ -100,6 +109,15 @@ export const storage = {
       }
       return value;
     },
+    notifications: async () => {
+      const { value } = await Preferences.get({
+        key: StorageKey.Notifications,
+      });
+      if (!value) {
+        throw new Error(`Can't find '${StorageKey.Notifications}' in storage`);
+      }
+      return value === "true";
```

Comment:
Is it possible that some notification in the storage will be false?
Looks like this check is unnecesaary